internal/webhook/envtestutil/envtestutil.go

```diff
@@ -123,7 +123,7 @@ func NewDSC(name, namespace string, opts ...func(*dscv1.DataScienceCluster)) *ds
 	dsc := &dscv1.DataScienceCluster{
 		TypeMeta: metav1.TypeMeta{
 			Kind:       gvk.DataScienceCluster.Kind,
-			APIVersion: gvk.DataScienceCluster.Version,
+			APIVersion: dscv1.GroupVersion.String(),
```

Comment:
This `.String()` has been reverted to `.Version`, but both are fine. It will be integrated later.

Reply:
Yeah, I fixed that one, but I'm not sure why it was  `gvk.DataScienceCluster.Version` before. Let's leave it then.